src/arkit.cs

```diff
@@ -2004,9 +2017,13 @@ interface ARSkeleton2D {
 		[Protected, Export ("jointLandmarks")]
 		IntPtr RawJointLandmarks { get; }
 
+		[EditorBrowsable (EditorBrowsableState.Advanced)]
 		[Export ("landmarkForJointNamed:")]
 		[MarshalDirective (NativePrefix = "xamarin_simd__", Library = "__Internal")]
-		Vector2 GetLandmarkPoint ([BindAs (typeof (ARSkeletonJointName))] NSString jointName);
+		Vector2 GetLandmarkPoint (NSString jointName);
+
+		[Wrap ("GetLandmarkPoint (jointName.GetConstant()!)")]
+		Vector2 GetLandmarkPoint (ARSkeletonJointName jointName);
```

Comment:
what was the reason to remove `[BindAs]` ?

Reply:
same